trunk/SUAVE/Methods/Noise/Fidelity_One/Propeller/propeller_noise_frequency.py

```diff
@@ -0,0 +1,327 @@
+            NOrder = m*blades_number    #order
+            Jmb    = scipy.special.jn(NOrder, Arg)        
+            sound_speed_ft = 1115.49
+           
+            A = 538673*m*blades_number*Mach_tip*np.sin(angle)/(dist_axis*propeller_diameter*convection_factor)
+           
+            B = (np.cos(angle)/convection_factor)*thrust-(550/(zeff*zeff*Mach_tip*Mach_tip*sound_speed_ft)*hpower)
+                                                               
+            SPL[m] = 20*np.log10(A*B*plm*Jmb)
+            
+         #   SPL1 = 20*np.log10(538673*m*blades_number*Mach_tip*np.sin(angle)/(dist_axis*propeller_diameter*convection_factor) \
+         #                     *((np.cos(angle)*thrust/convection_factor)-(550*hpower/(zeff*zeff*Mach_tip*Mach_tip*sound_speed_ft))) \
+         #                     *plm*Jmb)
+            
+           # print "A = ", A, "B = ", B,"plm = ",plm," bessel = ", Jmb, 
+         #   print " noise = ",SPL, "noise1 = ", SPL1
+    #*************************************************************************************************************************            
+    #************************************************************************************************************************* 
+    
+    #        print "total_pressure",total_pressure
+    #        print "total_monopole",total_monopole
+    #        print "total_drag",total_drag
+    #        print "total_lift",total_lift
+            
+        pt_total[jj]=total_pressure
+        pt_monopole[jj]=total_monopole
+        pt_drag[jj]=total_drag
+        pt_lift[jj]=total_lift
+        
+        p_2_total = (total_pressure*total_pressure) + p_2_total
+        p_2_monopole = (total_monopole*total_monopole) + p_2_monopole
+        p_2_drag = (total_drag*total_drag) + p_2_drag
+        p_2_lift = (total_lift*total_lift) + p_2_lift
+        
+       # print p_time[time]
+        
+        total_pressure = 0.0
+        total_monopole = 0.0
+        total_drag = 0.0
+        total_lift = 0.0    
+        
+       
+    
+    p_rms_total = np.sqrt(np.mean(p_2_total))
+    p_rms_monopole = np.sqrt(np.mean(p_2_monopole))
+    p_rms_drag = np.sqrt(np.mean(p_2_drag))
+    p_rms_lift = np.sqrt(np.mean(p_2_lift))
+    
+    # pack
+    noise_result = Data()
+    noise_result.p_rms_total = p_rms_total
+    noise_result.p_rms_monopole = p_rms_monopole
+    noise_result.p_rms_drag = p_rms_drag
+    noise_result.p_rms_lift = p_rms_lift
+    noise_result.SPL_total = 20*np.log10(p_rms_total/p_base)
+    noise_result.SPL_monopole = 20*np.log10(p_rms_monopole/p_base)
+    noise_result.SPL_drag = 20*np.log10(p_rms_drag/p_base)
+    noise_result.SPL_lift = 20*np.log10(p_rms_lift/p_base)    
+    noise_result.time = timej
+    noise_result.pt_total=pt_total
+    noise_result.pt_monopole=pt_monopole
+    noise_result.pt_drag=pt_drag
+    noise_result.pt_lift=pt_lift
+    
+    return (noise_result)
+
+    #print "SPL_total =", 20*np.log10(p_rms_total/p_base)
```

Comment:
Remove comments

Reply:
I removed this file from this PR and will put it in a noise PR